cypress/e2e/eholdings/package/eholdings-packages-export-w-long-note.cy.js

```diff
@@ -0,0 +1,165 @@
+        Permissions.uiAgreementsSearch.gui,
+        Permissions.uiNotesAssignUnassign.gui,
+        Permissions.uiNotesItemCreate.gui,
+        Permissions.uiNotesItemView.gui,
+        Permissions.exportManagerAll.gui,
+      ]).then((userProperties) => {
+        testData.user = userProperties;
+
+        AssignedUsers.assignUserToDefaultCredentialsViaApi({ userId: testData.user.userId });
+
+        cy.login(testData.user.username, testData.user.password, {
+          path: `${TopMenu.eholdingsPath}?searchType=packages`,
+          waiter: EHoldingsPackages.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken().then(() => {
+        FileManager.deleteFileFromDownloadsByMask(`*${testData.package.id}_package.csv`);
+        FileManager.deleteFileFromDownloadsByMask(testData.packageData);
+        FileManager.deleteFileFromDownloadsByMask(testData.titleData);
+        Notes.deleteNotesForEHoldingViaApi(testData.package.id);
+        Users.deleteViaApi(testData.user.userId);
+      });
+    });
+
+    it(
+      'C357529 Export all "Titles" (less than 10k) of "Package" record with large "Note" (around 4k symbols). (spitfire) (TaaS)',
+      { tags: ['criticalPath', 'spitfire'] },
+      () => {
+        // Fill in the input field with the search query, Click on the "Search" button.
+        EHoldingsPackagesSearch.byName('Cambridge');
+        EHoldingsPackages.verifyListOfExistingPackagesIsDisplayed();
+
+        // Click on the "Selection status" accordion, Click on the "Selected" status.
+        EHoldingsPackagesSearch.bySelectionStatus(testData.package.status);
+        EHoldingsPackages.verifyOnlySelectedPackagesInResults();
+
+        // Open "Package" record which has titles not more than 10 000, and not less than 9 000.
+        EHoldingsPackages.sortPackagesByTitlesCount({ minTitlesCount: 9000 }).then((packages) => {
+          testData.package.id = packages[0].id;
+          testData.package.name = packages[0].name;
+          testData.package.titles = packages[0].count;
+
+          testData.packageData = `package_data_${testData.package.id}.csv`;
+          testData.titleData = `title_data_${testData.package.id}.csv`;
+
+          // View "Package" record with "Total titles" value more than 1.
+          EHoldingsPackages.openPackageWithExpectedName(testData.package.name);
+          EHoldingsPackageView.verifyPackageDetailViewIsOpened(
+            testData.package.name,
+            testData.package.titles,
+            testData.package.status,
+          );
+
+          // Verify "Package" record doesn't have assigned "Notes" records.
+          EHoldingsPackageView.checkNotesSectionContent();
+
+          // Click on the "New" button under "Notes" accordion.
+          const NoteEditForm = EHoldingsPackageView.openAddNewNoteForm();
+
+          // Fill in the "General information" of new note.
+          NoteEditForm.fillNoteFields(testData.note);
```

Comment:
Missed Step 7

Reply:
Discussed with QA Team: step with text styling adjustment can be skipped in autotest.